frontend/src/components/InlineMessage/InlineMessage.tsx

```diff
@@ -1,12 +1,10 @@
-import { useMemo } from 'react'
 import ReactMarkdown from 'react-markdown'
-import { Components } from 'react-markdown/src/ast-to-react'
 import { Box, Flex, Icon, useMultiStyleConfig } from '@chakra-ui/react'
 
 import { BxsErrorCircle, BxsInfoCircle } from '~/assets/icons'
 import { InlineMessageVariant } from '~/theme/components/InlineMessage'
 
-import Link from '../Link'
+import { useMdComponents } from '../../hooks/useMdComponents'
```

Comment:
nit: should we use the alias path instead?

```suggestion
import { useMdComponents } from '~hooks/useMdComponents'
```